lib/modules/dosomething/dosomething_global/dosomething_global.module

```diff
@@ -6,6 +6,19 @@
 
 include_once 'dosomething_global.features.inc';
 define('GLOBAL_ADMIN_ROLES', 'mexico admin,brazil admin');
+define('GLOBAL_ADMIN_LANGUANGES', serialize(['es-mx' => 'mexico admin', 'pt-br' => 'brazil admin']));
```

Comment:
Yeah, that's what I was thinking, ok to merge this as is and create an issue for the strongarm lang switching?